### DISCUSSION TITLE
Natural frequencies

### DISCUSSION THREAD

**Discussion by hserdogan94** [petrobras/ross#1094] [Q&A] [answered]
First of all, thank you for this great software.

I have a question about natural frequency. I'm trying to model a free-free shaft to determine its natural frequencies. However, after creating the code snippet, I noticed that every time I run the code, I obtain different natural frequencies for the same shaft configuration. I believe this isn't normal. Does anyone know what might be causing this issue or how to resolve it?

Thank you.

**Comment by jguarato** [comment]
Hi @hserdogan94!

Are you using the ross version installed with pip (latest release version) or a version of the code from the main branch of the ross repository (development version)?

**Reply by hserdogan94** [reply]
Hi @jguarato ,

Thank you for your response. I am using pip version.

**Reply by jguarato** [reply, marked as answer]
Could you test the development version to see if this issue persists?

To install the development version:
```python
pip install git+https://github.com/petrobras/ross.git
```

**Reply by hserdogan94** [reply]
I tried it right now. It is more stable (enough stable I mean just a few Hz) and I think I obtained the results matching experimental values. I am so happy :) 

Thank you so much!

**Reply by jguarato** [reply]
You are welcome! I'm glad I could help.

We'll keep investigating to improve this part. 
If you encounter other issues, we recommend reporting them as new [Issues](https://github.com/petrobras/ross/issues).
